### ISSUE TITLE
why libclang?

### ISSUE THREAD

**Issue by soloturn** [mikedilger/gossip#647]
to compile with ffmpeg support for arch linux, why it would need libclang, or in other words, what is in libclang for this purpose?

**Comment by mikedilger** [comment]
It just needed it on my system, or someone else's system. If you can get it working without libclang, do that.

But I'm considering ripping out the ffmpeg/video stuff because it is very difficult to match up to each person's different system, and has caused bugs for me, freezing my client.  I never run into these kinds of freezing/crashing bugs in rust code, but C code like ffmpeg is plagued with them, and there is no video player written in rust AFAIK.

**Comment by dreirund** [comment]
> *But I'm considering ripping out the ffmpeg/video stuff*

Can't you just use external players via the command line?

**Comment by mikedilger** [comment]
Yes we can but the UX is different.  We have a weblink to watch the video in your browser, that won't be going away.
